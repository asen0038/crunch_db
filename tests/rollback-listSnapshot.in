set a 2 3 5
snapshot
set a 8 9 0
get a
rollback 1
get a
list snpshots
bye
