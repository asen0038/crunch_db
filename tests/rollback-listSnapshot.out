>
ok

>
saved as snapshot 1

>
ok

>
[8 9 0]

>
ok

>
[2 3 5]

>
1

>
bye
